src/epics/login/silentLogin.epic.ts

```diff
@@ -27,17 +27,17 @@ export const silentLogin = (action$: AnyAction, state$: any) => action$.pipe(
         .pipe(
             switchMap(() => userService.getUser()
                 .pipe(
-                    map(authActions.loginSuccess),
                     tap(() => console.log('Silent login success using authToken')),
+                    mergeMap((data) => [authActions.loginSuccess(data), authActions.refreshAuthStart(), userActions.getProjectList()]),
```

Comment:
I would actually not perform the auth token refresh here. The idea what we talked about was to fire an event when the application started that would automatically check if it needed to refresh the token, do so if needed and set a timer (as you did)